babel.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
+        },
+      },
+    ],
+    ["@babel/preset-react", {"runtime": "automatic"}]
+  ],
+};
```

Comment:
- [ ] This file appears to have more than it should, kindly follow the steps to letter and I am sure you will fix this.
The following is what is supposed to be there
```
module.exports = {
  presets: [
    [
      '@babel/preset-env',
      {
        targets: {
          node: 'current',
        },
      },
    ],
    '@babel/preset-react',
  ],
};
```